From ea9fba16d972becc84cd2a82d25030975dc609a5 Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Sat, 30 Sep 2023 13:09:27 +0200
Subject: [PATCH] ARM: dts: qcom: ipq4019: add label to SCM

Some IPQ4019 boards require SDI to be disabled by adding a property to the
SCM node, so lets make that easy by adding a label to the SCM node.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 arch/arm/boot/dts/qcom/qcom-ipq4019.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/qcom/qcom-ipq4019.dtsi
+++ b/arch/arm/boot/dts/qcom/qcom-ipq4019.dtsi
@@ -155,7 +155,7 @@
 	};
 
 	firmware {
-		scm {
+		scm: scm {
 			compatible = "qcom,scm-ipq4019", "qcom,scm";
 		};
 	};
